packages/components/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import type { CommonProps } from '@contentful/f36-core';
+
+import { getTabsStyles } from './Tabs.styles';
+export interface TabsProps extends CommonProps {
+  variant?: 'default' | 'horizontal' | 'vertical';
+  role?: 'navigation' | 'tablist';
+  children?: React.ReactNode;
+}
+
+export function Tabs({
+  className,
+  children,
+  variant = 'default',
+  testId = 'cf-ui-tabs',
+  role = 'tablist',
+  style,
+}: TabsProps): React.ReactElement {
+  const styles = getTabsStyles({ className, variant });
+
+  const elementProps = {
+    'data-test-id': testId,
```

Comment:
Ideally, we would use `Box` component inside, and just pass `testId={testId}` without providing a specific data attribute here.